packages/neo4j-driver/src/driver.js

```diff
@@ -67,7 +67,7 @@ class Driver extends CoreDriver {
     return new RxSession({
       session: this._newSession({
         defaultAccessMode,
-        bookmarks,
+        bookmarkOrBookmarks: bookmarks,
```

Comment:
Interesting decision to rename `bookmarks` internally (admittedly `bookmarkOrBookmarks` is the more precise name). :shrug:
Definitely out of scope of this PR.